Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -11,36 +11,45 @@ using namespace std;
 
 void foo(int *a, int b);
 void bar(int *a, int b);
+void MySwap(int *a, int *b);
 
 int main()
 {
     int x = 3;
     int y = 4;
     int *p = &x;
+    int *q= &y;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: adress of the variable p is pointing to (adress of x)" << endl;
```

Comment:
Please be consistent and place your prediction within `<< <<` like the instruction says.